packages/clerk-js/src/ui/components/SignIn/SignInFactorOne.tsx

```diff
@@ -29,7 +34,7 @@ const factorKey = (factor: SignInFactor | null | undefined) => {
   return key;
 };
 
-const isNotResetPasswordStrategy = (strategy: SignInStrategy) => strategy !== 'reset_password_code';
+const isNotResetPasswordStrategy = (strategy: SignInStrategy) => !strategy.includes('reset_password');
```

Comment:
🔧 Let's be more specific
```suggestion
const resetPasswordStrategies = ['reset_password_email_code', 'reset_password_phone_code']
const isNotResetPasswordStrategy = (strategy: SignInStrategy) => !resetPasswordStrategies.includes(strategy);
```